components/MarkdigUI/src/MultiTarget.props

```diff
@@ -0,0 +1,9 @@
+<Project>
+    <PropertyGroup>
+        <!--
+          MultiTarget is a custom property that indicates which target a project is designed to be built for / run on.
+          Used to create project references, generate solution files, enable/disable TargetFrameworks, and build nuget packages.
+        -->
+        <MultiTarget>uwp;wasdk;wpf;wasm;linuxgtk;macos;ios;android;</MultiTarget>
```

Comment:
Are you planning to support Uno.WASM as well? If not, you can remove everything after `uwp;wasdk;`

Might make it easier for initial testing as well :)